llvm/test/Transforms/PGOProfile/ctx-instrumentation.ll

```diff
@@ -250,6 +250,25 @@ define void @no_counters() {
   call void @bar()
   ret void
 }
+
+define void @inlineasm() {
```

Comment:
I'm not sure what specifically we are looking for below with regards to the check affected by this patch. Maybe add a comment.

Reply:
done.

Reply:
But isn't the "llvm.instrprof.increment" instrumentation? Oh I see we are looking to make sure there is no "llvm.instrprof.callsite" instrumentation, right? Maybe just mention that.

Reply:
Yup - and clarified.